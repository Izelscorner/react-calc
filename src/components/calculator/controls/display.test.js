import React from 'react';
import { shallow } from 'enzyme';
import Display from './display';
import toJson from 'enzyme-to-json';

describe('<Display />', () => {
  let wrapper;

  beforeAll(() => {
    wrapper = shallow(<Display display={10}  />);
  });

  it('matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('renders the label and value', () => {
    const label = wrapper.find("label");
    expect(label.text()).toEqual("10");
  });

});
