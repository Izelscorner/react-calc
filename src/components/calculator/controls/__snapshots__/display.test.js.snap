// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display /> matches snapshot 1`] = `
<div
  className="display"
>
  <div
    className="display-branding"
  />
  <label>
    10
  </label>
</div>
`;
